cluster/aks/composition.yaml

```diff
@@ -1,20 +1,27 @@
 apiVersion: apiextensions.crossplane.io/v1
 kind: Composition
 metadata:
-  name: xaks.azure.platformref.crossplane.io
+  name: xaks.azure.platformref.upbound.io
   labels:
     provider: azure
 spec:
   compositeTypeRef:
-    apiVersion: azure.platformref.crossplane.io/v1alpha1
+    apiVersion: azure.platformref.upbound.io/v1alpha1
     kind: XAKS
   resources:
     - base:
-        apiVersion: compute.azure.crossplane.io/v1alpha3
-        kind: AKSCluster
+        apiVersion: containerservice.azure.upbound.io/v1beta1
+        kind: KubernetesCluster
         spec:
-          location: West US 2
-          version: "1.21.7"
+          forProvider:
+            location: West US 2
+            kubernetesVersion: "1.24.3"
```

Comment:
is this a version that has all the server side CRD perf fixes?

Reply:
Great question, from what I can tell yes, https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#changelog-since-v1243
The changelog contains https://github.com/kubernetes/kubernetes/pull/111648
Also, it is the highest available AKS version at the moment
```
az aks get-versions --location westus2 --output table
KubernetesVersion    Upgrades
-------------------  -----------------------
1.24.3               None available
1.24.0               1.24.3
1.23.8               1.24.0, 1.24.3
1.23.5               1.23.8, 1.24.0, 1.24.3
1.22.11              1.23.5, 1.23.8
1.22.6               1.22.11, 1.23.5, 1.23.8
```